app/models/activity.rb

```diff
@@ -534,15 +534,22 @@ def comments_for_report(report_id:)
   end
 
   def requires_collaboration_type?
```

Comment:
Do you think we could have some tests for these? Particularly where we have multiple conditions, it always feels good to have this covered and documented in a unit test.